src/metabase/api/user.clj

```diff
@@ -242,7 +242,10 @@
 (defn- maybe-add-sso-source
   "Adds `sso_source` key to the `User`, so FE could determine if the user is logged in via SSO."
   [{:keys [id] :as user}]
-  (if (premium-features/enable-sso?)
+  (if (or (premium-features/enable-sso-jwt?)
+          (premium-features/enable-sso-saml?)
+          (premium-features/enable-sso-ldap?)
+          (premium-features/enable-sso-google?))
```

Comment:
maybe define a `premium-features/enable-any-sso? instead of having these blocks of 4 predicates?

Reply:
Done.